glib/src/subclass/mod.rs

```diff
@@ -87,11 +90,7 @@
 //!         // Called every time a new instance is created. This should return
 //!         // a new instance of our type with its basic values.
 //!         fn new() -> Self {
```

Comment:
you could just drop the new method as well, it defaults to calling `Self::default()` already, but that might make it unclear how all of that works